module/Activity/src/Activity/Service/Activity.php

```diff
@@ -132,7 +133,7 @@ public function createUpdateProposal(ActivityModel $oldActivity, array $params,
             $em->remove($oldUpdate);
             $em->flush();
 
-            return $oldProposal;
+            return false;
```

Comment:
If an admin edits an approved activity without pending updates, this will return false, so it is not possible to immediately edit and approve that edit, which I think is intended by this PR.